Public/Remove-WinADDuplicateObject.ps1

```diff
@@ -24,6 +24,9 @@
     $Count = 0
     $DuplicateObjects = Get-WinADDuplicateObject @getWinADDuplicateObjectSplat
     foreach ($Duplicate in $DuplicateObjects | Select-Object -First $LimitProcessing) {
+        If($Duplicate.ProtectedFromAccidentalDeletion -eq "True"){
+            Get-ADObject -Filter {ObjectGUID -eq "$($Duplicate.ObjectGUID)"} -ErrorAction Stop | Set-ADObject -ProtectedFromAccidentalDeletion $false
```

Comment:
You need Error Action on Set-AdObject as well, and you need to have try/catch logic for that and decide what happens next. My guess would be - if this happens you need to continue the loop for next object, and not let Remove part to happen.

Reply:
Added error action for Set-AdObject, also added try catch to handle error scenarios.